lib/parallel_tests/tasks.rb

```diff
@@ -56,10 +56,10 @@ def check_for_pending_migrations
         end
       end
 
-      # parallel:spec[:count, :pattern, :options]
+      # parallel:spec[:count, :pattern, :options, :serialize_stdout]
```

Comment:
I don't think serialize_stdout is important enough to be the 4th argument, prefer using the binary instead ?